CONTRIBUTING.md

```diff
@@ -30,43 +30,36 @@ style configurations are commonly useful. For IntelliJ 14, copy the style to
-  address it as a separate one or multiple commits with a descriptive commit
-  message (try avoid using vauge `addressed pr feedback` type of messages).
-
-  Project maintainers are obligated to squash those commits into one when
-  merging.
-
 ## Guidelines for Pull Requests
 How to get your contributions merged smoothly and quickly.
  
 - Create **small PRs** that are narrowly focused on **addressing a single concern**. We often times receive PRs that are trying to fix several things at a time, but only one fix is considered acceptable, nothing gets merged and both author's & review's time is wasted. Create more PRs to address different concerns and everyone will be happy.
  
-- For speculative changes, consider opening an issue and discussing it first. If you are suggesting a behavioral or API change, consider starting with a [gRFC proposal](https://github.com/grpc/proposal). 
- 
-- Provide a good **PR description** as a record of **what** change is being made and **why** it was made. Link to a github issue if it exists.
- 
-- Don't fix code style and formatting unless you are already changing that line to address an issue. PRs with irrelevant changes won't be merged. If you do want to fix formatting or style, do that in a separate PR.
- 
-- Unless your PR is trivial, you should expect there will be reviewer comments that you'll need to address before merging. We expect you to be reasonably responsive to those comments, otherwise the PR will be closed after 2-3 weeks of inactivity.
- 
-- Maintain **clean commit history** and use **meaningful commit messages**. See [maintaining clean commit history](#maintaining-clean-commit-history) for details.
- 
+- For speculative changes, consider opening an issue and discussing it to avoid
+  wasting time on an inappropriate approach. If you are suggesting a behavioral
+  or API change, consider starting with a [gRFC
+  proposal](https://github.com/grpc/proposal).
+
+- Follow [typical Git commit message](https://cbea.ms/git-commit/#seven-rules)
+  structure. Have a good **commit description** as a record of **what** and
+  **why** the change is being made. Link to a GitHub issue if it exists. The
+  commit description makes a good PR description and is auto-copied by GitHub if
+  you have a single commit when creating the PR.
+
+  If your change is mostly for a single module (other module changes are
+  trivial), prefix your commit summary with the module name changed. Instead of
+  "Add HTTP/2 faster-than-light support to gRPC Netty" it is more terse as
+  "netty: Add faster-than-light support".
+
+- Don't fix code style and formatting unless you are already changing that line
+  to address an issue. If you do want to fix formatting or style, do that in a
+  separate PR.
+
+- Unless your PR is trivial, you should expect there will be reviewer comments
+  that you'll need to address before merging. Make review changes as additional
```

Comment:
"review changes" is nice and short, but feels like someone could get confused what it means (though the more I think about it, the less likely it seems).  "changes in response to reviewer comments" might be too verbose.

Reply:
Hmm... yeah, I see what you mean. "Address comments with additional commits..."? "Use additional commits for changes so the..."? "for needed changes"? "for necessary changes"? Does "changes" need an adjective?